scripts/templates/webdriver.tpl.d.ts

```diff
@@ -391,6 +391,61 @@ declare namespace WebDriver {
         sessionId: string;
     }
 
+    // object with no match
+    interface ProtocolCommandResponse {
+        [key: string]: any;
+    }
+
+    // webdriver.json
+    interface SessionReturn extends DesiredCapabilities {
+        // addtional capabilities return by remote
+        [key: string]: any;
+    }
+
+    interface StatusReturn {
+        ready?: boolean,
+        message?: string,
+        [key: string]: any
+    }
+
+    interface WindowHandle {
+        handle: string,
+        type: string
+    }
+
+    interface RectReturn {
+        x: number,
+        y: number,
+        width: number,
+        height: number
+    }
+    // appium.json
+
+    interface StringsReturn {
```

Comment:
Can you please have only one interface for `[key:string]: string` and extended other interfaces?
ex:
`interface SessionReturn extends DesiredCapabilities, ProtocolCommandResponse { }`

Reply:
Done

Reply:
Can we remove this one and use just ProtocolCommandResponse ?

Reply:
The value here is `string`, while `any` in `ProtocolCommandResponse`.